docs/devel/development.md

```diff
@@ -50,6 +50,14 @@ Official releases are built using Docker containers. To build Kubernetes using
 Docker please follow [these
 instructions](http://releases.k8s.io/HEAD/build/README.md).
 
+### Local OS/shell environment
+
+Many of the Kubernetes development helper scripts rely on a fairly up-to-date GNU tools
```

Comment:
Would it be better to just have hack/verify-godeps.sh do a version check for diff? Then it's not a documentation issue but one that the script will find on its own and inform the user.